tests/email_framework/test_performance.py

```
@@ -0,0 +1,353 @@
+        """
+        
+        large_css = """
+        .container { padding: 20px; background: #f5f5f5; }
+        .item { margin: 10px; padding: 15px; border: 1px solid #ddd; }
+        """ + "\n".join([f'.item-{i} {{ color: #{i:03x}; }}' for i in range(100)])
+        
+        with patch('microsoft_mcp.email_framework.css_inliner.inline_css') as mock_inline:
+            mock_inline.return_value = large_html.replace('class="', 'style="')
+            
+            start_time = time.time()
+            result = mock_inline(large_html, large_css)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)